autosklearn/pipeline/components/feature_preprocessing/truncatedSVD.py

```diff
@@ -39,6 +39,7 @@ def get_properties(dataset_properties=None):
                 'handles_classification': True,
                 'handles_multiclass': True,
                 'handles_multilabel': True,
+                'handles_multioutput': False,
```

Comment:
Hey, truncatedSVD shouldn't look at the y values, so it should be able to handle multi-output regression.

Reply:
Yeah, I think I kind of missed these two, I've updated them. Thanks!